packages/nextjs/src/config/loaders/wrappingLoader.ts

```diff
@@ -157,8 +157,6 @@ export default function wrappingLoader(
       .replace(/(.*)/, '/$1')
       // Pull off the file name
       .replace(/\/[^/]+\.(js|ts|jsx|tsx)$/, '')
-      // Remove routing groups: https://beta.nextjs.org/docs/routing/defining-routes#example-creating-multiple-root-layouts
-      .replace(/\/(\(.*?\)\/)+/g, '/')
```

Comment:
Sneaking this in. This is actually a bug because routing groups should part of the txn name. I added this when I didn't fully understand the app router yet.